tests/iot_config.h

```diff
@@ -121,7 +121,7 @@
 
 /* Enable asserts in the libraries. */
 #define IOT_CONTAINERS_ENABLE_ASSERTS           ( 1 )
-#define IOT_MQTT_ENABLE_ASSERTS                 ( 1 )
+#define IOT_MQTT_ENABLE_ASSERTS                 ( ! IOT_TEST_COVERAGE )
```

Comment:
So is this change just disabling asserts for unit test ?

Reply:
Only when building for coverage.